dashboard/test/ui/features/SectionActionDropdown.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Using the SectionActionDropdown
+  Scenario: Viewing progress from SectionActionDropdown
+    Given I create a teacher-associated student named "Sally"
+    And I give user "Teacher_Sally" hidden script access
+    And I complete the level on "http://studio.code.org/s/allthethings/stage/2/puzzle/1"
+    And I complete the free response on "http://studio.code.org/s/allthethings/stage/27/puzzle/1"
+    And I submit the assessment on "http://studio.code.org/s/allthethings/stage/33/puzzle/1"
+    And I sign out
+
+    When I sign in as "Teacher_Sally"
+    And I am on "http://code.org/teacher-dashboard?no_home_redirect=1"
```

Comment:
What is this testing?  I think http://code.org/teacher-dashboard?no_home_redirect=1 is basically an unsupported page now - can we bypass it and go directly to the "Student Accounts and Progress" page instead?